src/node/internal/crypto.d.ts

```diff
@@ -1,5 +0,0 @@
-// Copyright (c) 2017-2022 Cloudflare, Inc.
-// Licensed under the Apache 2.0 license found in the LICENSE file or at:
-//     https://opensource.org/licenses/Apache-2.0
-
-export function getRandomInt(min: number, max: number): number;
```

Comment:
Nit: Please don't delete this file. We'll have PRs coming that actually fill this out more with actual content in the coming couple of weeks.